test/e2e/DCATokenDescriptor/token-descriptor-is-valid.spec.ts

```diff
@@ -60,22 +60,36 @@ contract('DCATokenDescriptor', () => {
 
   it('Validate tokenURI result', async () => {
     // Deposit
-    const response = await DCAPair.deposit(tokenA.address, tokenA.asUnits(10), 20, swapInterval);
+    const response = await DCAPair.deposit(tokenA.address, tokenA.asUnits(10), 2, swapInterval);
```

Comment:
Is this change ok? Why are we chaning 20 to 2?

Reply:
We are changing the amount of swaps from 20 to 2. So we can make swapsLeft be 0 only with two swaps